src/Datadog.Trace.ClrProfiler.Native/cor_profiler.h

```diff
@@ -21,6 +21,7 @@ class CorProfiler : public CorProfilerBase {
   bool is_attached_ = false;
   RuntimeInformation runtime_information_;
   std::vector<Integration> integrations_;
```

Comment:
You should be able to remove the `integrations_` member now. I think it was only used to carry the list of integration to the ModuleLoadFinished event

Reply:
Right!, done in: https://github.com/DataDog/dd-trace-dotnet/pull/831/commits/beb82bdd0c481b19636eccbce21614c1ab78068c